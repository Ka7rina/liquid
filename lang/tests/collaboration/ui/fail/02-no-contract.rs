use liquid_lang as liquid;

#[liquid::collaboration]
mod noop {}

fn main() {}