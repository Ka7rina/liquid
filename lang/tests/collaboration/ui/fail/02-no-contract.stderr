error: no `#[liquid(contract)]` struct found in this collaboration
 --> $DIR/02-no-contract.rs:4:1
  |
4 | mod noop {}
  | ^^^^^^^^^^^
